packages/code-du-travail-modeles/src/modeles/conventions/commerces_de_detail_non_alimentaires.yaml

```
@@ -4,4 +4,50 @@ contrat salarié . convention collective . commerces de detail non alimentaires:
   valeur: oui
   cdtn:
     idcc: 1517
-    préavis-retraite: false
+    préavis-retraite: true
+
+contrat salarié . convention collective . commerces de detail non alimentaires . catégorie professionnelle:
+  titre: Catégorie professionnelle
+  question: Quelle est la catégorie professionnelle du salarié ?
+  description: La catégorie professionnelle du salarié est habituellement mentionnée sur le **bulletin de salaire**.
```

```suggestion
  description: La catégorie professionnelle du salarié est habituellement mentionnée sur le <b>bulletin de salaire</b>.
```

---

Ah non pas de <b> please
```suggestion
  description: La catégorie professionnelle du salarié est habituellement mentionnée sur le <strong>bulletin de salaire</strong>.
```